bin/BuildPackages.sh

```diff
@@ -186,6 +186,11 @@ run_configure_and_make() {
   if [[ -x autogen.sh && ! -x configure ]]
   then
     ./autogen.sh
+    # (Hopefully) temporary fix for Autoconf 2.71
```

Comment:
We can try to simply update config.sub and config.guess, that's something we have to do occasionally anyway (see issue #1478). There are other changes to be made in configure.ac. I'll look into that. But all of this must keep working with autoconf 2.69, which is what almost everyone has. In a quick test of mine, it does so.

Reply:
I think the issue was actually specific to the io package, which actually had a buggy configure.ac that happened to work in autoconf 2.69, and now 2.70 detected it. To be precise: I set it up to not check for a "canonical host type", but then used autoconf macros which actually require that; in autoconf 2.69 this sailed through (and possibly resulted in buggy behavior, though this unlikely on any modern day OS). In autoconf 2.70 it detects this requirement and computes the canonical host type, which require config.guess and config.sub ... hence the issue